<!-- -->
<launch>

	<arg name="sim_mode" default="False"/>
	<arg name="linear_interpolate_mode" default="True"/>
	<!-- rosrun waypoint_follower pure_pursuit -->
	<node pkg="waypoint_follower" type="pure_pursuit" name="pure_pursuit" output="screen">
		<param name="sim_mode" value="$(arg sim_mode)"/>
		<param name="linear_interpolate_mode" value="$(arg linear_interpolate_mode)"/>
	</node>

</launch>
